Modules/_io/fileio.c

```diff
@@ -12,9 +12,6 @@
 #ifdef HAVE_SYS_TYPES_H
 #  include <sys/types.h>
 #endif
-#ifdef HAVE_SYS_STAT_H
```

Comment:
This include is covered by the `pycore_fileutils.h` include earlier, this code only uses `_Py_fstat_noraise`

Reply:
I don't think that this include is causing any issue. I may even prefer to keep it for S_ISCHR().